src/Webform.js

```diff
@@ -1356,6 +1356,7 @@ export default class Webform extends NestedDataComponent {
     let isChangeEventEmitted = false;
     // For any change events, clear any custom errors for that component.
     if (changed && changed.component) {
+      this.clearServerErrors();
```

Comment:
That means if component A has server error and we change any other component in the form (e.g. component B that is valid), the error from component A disappears. This behavior does not seem to me correct. Only if change happens for the component with server error, the error should disappear and only from the changed component. If there are some other components in the form with server errors, those errors should persist until those components are changed.

Reply:
Additionally please rebase PR